deepspeed/__init__.py

```diff
@@ -122,6 +126,7 @@ def initialize(args=None,
                                  mpu=mpu,
                                  dist_init_required=dist_init_required,
                                  collate_fn=collate_fn,
+                                 config=config,
```

Comment:
Do `DeepSpeedEngine` and `PipelineEngine` still need `config_params` though? Are those public APIs or internal? If they are internal then perhaps it's enough to alias `config_params` to `config` in `initialize` and adjust those 2 classes to receive just `config` ?

Reply:
Was contemplating this, if preferred I can move the logic to do this into the `initialize` function instead of doing it internally! Wanted to keep `initialize` as clean as possible, but didn't know which would be preferred

Reply:
Oh, I see, this is perfectly fine then. This is good. Thank you!